massa-models/src/execution.rs

```diff
@@ -0,0 +1,22 @@
+use crate::Slot;
```

Comment:
execution_*types*.rs?

Reply:
I don't think so, since it's arleady in `models` it means it will be some sort of "types".